src/Sentry.Maui.CommunityToolkit.Mvvm/MauiAppBuilderExtensions.cs

```diff
@@ -0,0 +1,32 @@
+using Sentry.Maui.CommunityToolkitMvvm;
```

Comment:
question: Namespace name:

```suggestion
using Sentry.Maui.CommunityToolkit.Mvvm;
```

Or is the "missing `.`" intentional?